### PR TITLE
Add https://www.iconfont.cn/

### PR DESCRIPTION
**Pull request by weiwill**
### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [x] This is Software as a Service not self hosted
 - [x] It has a free tier not just a free trial
 - [x] The submission mentions what is free
 - [x] The submission is not already present in the list
 - [x] The service has contact details of those running it and a privacy policy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1304,6 +1304,7 @@ Table of Contents
   * [framer.com](https://www.framer.com/) - Framer helps you iterate and animate interface ideas for your next app, website, or product—starting with powerful layouts. For anyone validating Framer as a professional prototyping tool: unlimited viewers, up to 2 editors, up to 3 projects.
   * [freeforcommercialuse.net](https://freeforcommercialuse.net/) — FFCU Worry-free model/property release stock photos
   * [Gradientos](https://www.gradientos.app) - Makes choosing a gradient fast and easy.
+  * [Iconfont](https://www.iconfont.cn/) – Download massive custom icons. Translating icons into Fonts. Project Collaboration Tools. Communication Platform.
```
Comment (ripienaar) [inline README.md:1307]: we can only add english sites to this list, is there a english version?

Comment (weiwill) [reply README.md:1307]: English translation is not complete

Comment (weiwill) [reply README.md:1307]: image

Comment (DreamOfIce) [reply README.md:1307]: > 我们只能将英文网站添加到此列表中，有英文版本吗？



> we can only add english sites to this list, is there a english version?

https://www.iconfont.cn/?lang=en-us

**Comment by weiwill** [comment]
@ripienaar

**Comment by FatGrizzly** [comment]
image
Still in Chinese

**Comment by DreamOfIce** [comment]
> image
> Still in Chinese

Look at the top navigation bar, a large part of the content has become English
h

**Comment by FatGrizzly** [comment]
> > image
> > Still in Chinese
> 
> Look at the top navigation bar, a large part of the content has become English h

We only accept sites that are fully English, not partial ones.

Please let us know once the full English translation is complete.

**Comment by weiwill** [comment]
Close it first, and then reopen it after there is a complete English translation

**Pull request closed without merging**